rafttest/interaction_env_handler_stabilize.go

```diff
@@ -25,7 +25,22 @@ import (
 )
 
 func (env *InteractionEnv) handleStabilize(t *testing.T, d datadriven.TestData) error {
-	idxs := nodeIdxs(t, d)
+	idxs := nodeIdxs(t, d) // skips key=value args
+	for _, arg := range d.CmdArgs {
```

Comment:
Never `datadriven` has `MaybeScanArgs`, but we haven't cut a release, so this is fine.

Reply:
Ok, thanks -- we can deal with that later.